charts/incubator/rtorrent-rutorrent-cross-seed/values.yaml

```diff
@@ -0,0 +1,96 @@
+image:
+  repository: ghcr.io/ac1dburnz/rtorrent-rutorrent-cross-seed
+  pullPolicy: IfNotPresent
+  tag: latest@sha256:018a0edda145c62fc6400a7de9d901a53c384d7fe6a4e928e0163c97c4bc8f1f
+
+service:
+  main:
+    ports:
+      main:
+        protocol: http
+        port: 8080
+  rtorrent:
+    enabled: true
+    ports:
+      rtorrent:
+        enabled: true
+        protocol: tcp
+        port: 33942
+  xmlrpc:
+    enabled: true
+    ports:
+      xmlrpc:
+        enabled: true
+        protocol: http
+        port: 8000
+  dht:
+    enabled: true
+    ports:
+      dht:
+        enabled: true
+        protocol: udp
+        port: 6881
+  webdav:
+    enabled: true
+    ports:
+      webdav:
+        enabled: true
+        protocol: http
+        port: 9000
+  rtinc:
+    enabled: true
+    ports:
+      rtinc:
+        enabled: true
+        port: 50000
+  crossseed:
+    enabled: true
+    ports:
+      crossseed:
+        enabled: true
+        protocol: tcp
+        targetPort: 2468
+        port: 2468
+
+persistence:
+  config:
+    enabled: true
+    mountPath: "/config"
+  data:
+    enabled: true
+    mountPath: "/data"
+    size: 256Gi
+  sessions:
+    enabled: true
+    mountPath: "/sessions"
+  cross-seed:
+    enabled: true
+    mountPath: "/root/.cross-seed"
+  passwd:
+    enabled: true
+    mountPath: "/passwd"
```

Comment:
Thats directory mount and not a file mount I suppose?

Reply:
Ya it’s a directory for the password files